azurerm/internal/services/applicationinsights/resource_arm_application_insights.go

```diff
@@ -166,10 +164,7 @@ func resourceArmApplicationInsightsCreateUpdate(d *schema.ResourceData, meta int
 
 	resp, err := client.CreateOrUpdate(ctx, resGroup, name, insightProperties)
 	if err != nil {
-		// @tombuildsstuff - from 2018-08-14 the Create call started returning a 201 instead of 200
-		// which doesn't match the Swagger - this works around it until that's fixed
-		// BUG: https://github.com/Azure/azure-sdk-for-go/issues/2465
-		if resp.StatusCode != http.StatusCreated {
+		if resp.StatusCode != http.StatusOK {
```

Comment:
we can remove this since the CreateOrUpdate method above checks this/returns an error if this isn't the case

```suggestion
```

Reply:
Done 👍